File: livingrimoire start here/LivinGrimoire java/src/Skills/logical/DiGamificationScouter.java
package Skills.logical;

import Auxiliary_Modules.AXGamification;
import Auxiliary_Modules.Responder;
import LivinGrimoire.Skill;

public class DiGamificationScouter extends Skill {
    private int lim = 2; // minimum for mood
    private final AXGamification axGamification;
    private final Responder noMood = new Responder("bored","no emotions detected","neutral");
    private final Responder yesMood = new Responder("operational", "efficient","mission ready","awaiting orders");

    public DiGamificationScouter(AXGamification axGamification) {
        this.axGamification = axGamification;
    }

    public void setLim(int lim) {
        this.lim = lim;
    }

    @Override
    public void input(String ear, String skin, String eye) {
        if (!ear.equals("how are you")){
            return;
        }
        if(axGamification.getCounter()>lim){
            setSimpleAlg(yesMood.getAResponse());
        }
        else{
            setSimpleAlg(noMood.getAResponse());
        }
    }
}

File: livingrimoire start here/LivinGrimoire Kotlin/kotlin/skills/logical/DiGamificationScouter.kt
package skills.logical

import auxiliary_modules.AXGamification
import auxiliary_modules.Responder
import livinGrimoire.DiSkillV2

class DiGamificationScouter(axGamification: AXGamification) : DiSkillV2() {
    private var lim = 2 // minimum for mood
    private val axGamification: AXGamification
    private val noMood: Responder = Responder("bored", "no emotions detected", "neutral")
    private val yesMood: Responder = Responder("operational", "efficient", "mission ready", "awaiting orders")

    init {
        this.axGamification = axGamification
    }

    fun setLim(lim: Int) {
        this.lim = lim
    }

    override fun input(ear: String, skin: String, eye: String) {
        if (ear != "how are you") {
            return
        }
        if (axGamification.counter > lim) {
            setSimpleAlg(yesMood.aResponse)
        } else {
            setSimpleAlg(noMood.aResponse)
        }
    }
}